chicken_app/src/Pages/Signup.js

```diff
@@ -43,11 +43,11 @@ function Signup(props) {
   };
 
   function areValidationErrors() {
-    let temp = {}
-    let errors = false
+    let temp = {} //allowing code to set a temporary value for the email/password
```

Comment:
Adding too many comments can be counterproductive. You can use variable names that are descriptive enough to speak for themselves. Temp is descriptive enough to use on it's own.